jena-base/src/main/java/org/apache/jena/atlas/lib/BitsLong.java

```diff
@@ -157,22 +157,35 @@ boolean test(long bits, long value, int start, int finish)
      *  @param start
      *  @param finish
      *  @return lon     */
-    
+
     public static final
     long access(long bits, int start, int finish)
     {
         check(start, finish) ;
-        return access$(bits, start, finish) ; 
+        return access$(bits, start, finish) ;
     }
-    
+
+    /**
+     * Clear the bit specified.
+     *  @param bits
+     *  @param bitIndex
+     *  @return long
+     */
+    public static final
+    long clear(long bits, int bitIndex)
+    {
+        check(bitIndex) ;
```

Comment:
Ditto above about `check` in `clear$`

Reply:
Can't find that one but what I have done is make the files exactly the same except s/int/long/. They now line up in diff and (easier to see!) line length.